crates/zksync/core/src/vm/farcall.rs

```diff
@@ -149,6 +201,137 @@ impl FarCallHandler {
         }
     }
 
+    /// Tracks the call stack for the actual transaction execution starting at `executeTransaction`.
```

Comment:
Does it mean we can't track anything in the bootloader or that we purposely chose not to?

Reply:
We chose not to, in order to be consistent with what EVM returns (and what we did with traces). If we were to record them, they'd mess up the depth calculation or require post-op depth adjustment/filtering (valid options) but for now the code is simpler if we simply ignore them.